api/v1alpha7/nnfcontainerprofile_webhook.go

```diff
@@ -135,6 +146,22 @@ func (r *NnfContainerProfile) validateContent() error {
 		if !workerOk || len(worker.Template.Spec.Containers) < 1 {
 			return fmt.Errorf("MPISpec.MPIReplicaSpecs.Worker must be present with at least 1 container defined")
 		}
+
+		// When it looks like we have a copy offload Launcher container, ensure the service account is correct
+		for _, c := range launcher.Template.Spec.Containers {
+			if isCopyOffloadContainer(c.Name) {
```

Comment:
if (it looks like it's not a copy-offload profile) {
  Hey, you don't get to use the copy-offload service account.
}

Reply:
Good call. Done.